incubator/identityservice/README.md

```diff
@@ -0,0 +1,158 @@
+# Identity Service
+
+Identity Service is used to provide single-sign-on and protect Web APIs in an ASP.NET Core application. An intuitive web UI dashboard is used to carry out configuration tasks for identity and authorization.
+
+## Features
+
+* Single-Sign-On (SSO) using OpenID connect
+* Protect Web APIs using the Client Credentials Flow
+* Manage users, claims and roles for ASP.NET Core Identity
+
+ For detailed instructions on how to use Identity Service, refer to the  [Identity Service Documentation](https://rcl-identityserver.github.io/documentation/)
+
+## Microsoft Azure Container Service (AKS) and ASP.NET Core
+
+Identity service is designed to be hosted in a Microsoft Azure Container Service (AKS) Kubernetes **Linux** cluster. It is geared towards developers who use ASP.NET Core to create web applications. Identity Service can be used as a gateway service to secure Web APIs in a micro services architecture. 
+
+## Prerequisites
+
+* Azure Container Service (AKS) Kubernetes Linux cluster
+* Azure SQL Database
+* SendGrid Email service
+
+## Installing the Chart
+
+### Azure AKS Cluster
+
+Create an AKS Linux cluster in the Azure portal or add Identity Service to an existing AKS Linux cluster. For detailed instructions on setting up an AKS cluster refer to the  [Azure AKS Documentation](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
+
+In the cluster, install the following if they are not already installed :
+
+* NGINX ingress controller
+* Helm
+* KUBE-LEO
```

Comment:
- Helm is a prerequisite anyways. No need to specify it here.
- Why is nginx-ingress a requirement? Can't you use any other ingress controller?
- kube-lego is deprecated. It shouldn't be required either.